tests/dataplicity/m2m/test_bencode.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8
+from dataplicity.m2m.bencode import encode, decode, EncodingError, DecodeError
+import pytest
+import six
+
+
+def test_bencode_encoder():
+    """ test bencoding capabilities
+    """
+    assert encode({}) == six.b('de')  # empty dict
+    assert encode({b'foo': 'bar'}) == six.b('d3:foo3:bare')
```

Comment:
I don't think we need `six.b`. We're not supporting anything before Py2.7.  `b'd3:foo3:bare'` should be fine.